python/pyspark/ml/tests.py

```diff
@@ -1843,6 +1844,28 @@ def tearDown(self):
 
 class ImageReaderTest(SparkSessionTestCase):
 
+    def test_ocv_types(self):
+        ocvList = ImageSchema.ocvTypes
+        self.assertEqual("Undefined", ocvList[0].name)
```

Comment:
confused, didn't you make this "N/A" instead of undefined - how does the test pass?

Reply:
Yes, good catch. There should have been if else clause for name and return Undefined for mode == -1.